packages/storefront/src/pages/component-v3-status.md

```diff
@@ -71,11 +71,11 @@ console.log(u, v);
 | ⛔ **Marque**                       | - Usage Docs missing                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
-| ✅ **Scroller**                     | - Usage Docs missing<br /> - property `gradientColorScheme` naming does not fit its type<br /> - uses custom button which should be shared / aligned<br /> - Button size needs different dimensions to fit every usecas (e.g. Tag)<br /> - click area is to small and should be enhanced<br /> - Mobile detection could be improved                                                                                                                                                                                                                |
+| ✅ **Scroller**                     | - Usage Docs missing<br /> - property `gradientColorScheme` naming does not fit its type<br /> - uses custom button which should be shared / aligned<br /> - Button size needs different dimensions to fit every usecas (e.g. Tag)<br /> - click area is to small and should be enhanced<br /> - Mobile detection could be improved<br /> - gradient / next + prev buttons in SSR not shown                                                                                                                                                        |
 | ✅ **Accordion**                    | - Usage Docs missing<br />- Optimize icon (size) for compact mode<br /> - Optimize hover state for default mode                                                                                                                                                                                                                                                                                                                                                                                                                                    |
 | ⛔ **Stepper Horizontal**           | - Usage Docs missing                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
Check is missing, perhaps additional comments, too.

Reply:
Was lost in master merge, good hint